### PR TITLE
Tabs styles

### PR DESCRIPTION
**Pull request by evgenoid**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[x] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```
Issue Number: GOOD-168

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

Screenshot 2024-05-14 at 15 24 08
Screenshot 2024-05-14 at 15 24 18

### CODE REVIEW

**Comment by evgenoid** [comment]
@ghsteff what we decided with this one?

**Comment by ghsteff** [comment]
> @ghsteff what we decided with this one?

So the tertiary tab variant looks pretty much the same as our button group which could be confusing if you use both of these in the same app. But this is a nice variant of tab to have pre-built 🤔

You know how we have some stories called `CustomTheme`? Maybe having this tertiary tab variant as a `CustomTheme` story would make the most sense because it's saying "This doesn't necessarily match our default theme, but it's one example of a way you could style this"

You think some designs would use all these tab variants in one app?

File: src/layout/Tabs/Tab.tsx [hunk 1]
```diff
@@ -49,6 +49,11 @@ export interface TabProps extends PropsWithChildren {
    */
   size?: 'small' | 'medium' | 'large';
 
+  /**
+   * The variant of the tab.
+   */
+  variant?: 'text' | 'button';
```
Comment (amcdnl) [inline src/layout/Tabs/Tab.tsx:55]: Whats this for ?

Comment (evgenoid) [reply src/layout/Tabs/Tab.tsx:55]: We have two different styles in figma:
image
image

Comment (amcdnl) [reply src/layout/Tabs/Tab.tsx:55]: And thats set internally?

Comment (evgenoid) [reply src/layout/Tabs/Tab.tsx:55]: Yes, for tertiary tabs variant

**Pull request closed without merging**